RecoTauTag/RecoTau/python/tools/runTauIdMVA.py

```diff
@@ -1314,11 +1345,12 @@ def getDeepTauVersion(self, file_name):
         """returns the DeepTau year, version, subversion. File name should contain a version label with data takig year \
         (2011-2, 2015-8), version number (vX) and subversion (pX), e.g. 2017v0p6, in general the following format: \
         {year}v{version}p{subversion}"""
-        version_search = re.search('(20[1,2][125678])v([0-9]+)(p[0-9]+|)[\\._]', file_name)
+        version_search = re.search('(20[1,2][125678]|RunII)v([0-9]+)(p[0-9]+|)[\\._]', file_name)
         if not version_search:
             raise RuntimeError('File "{}" has an invalid name pattern, should be in the format "{year}v{version}p{subversion}". \
                                 Unable to extract version number.'.format(file_name))
         year = version_search.group(1)
+        if year == 'RunII': year = 20161718
```

Comment:
I suppose this ID is trained with Run2 samples, so has it been tested to see if the performance still holds for Run3 samples?

Reply:
Yes, it was trained on mix of MC samples from all 2016-18 Run2 years to cope with limited statistics (I think mainly signal samples). It was checked that performance of the BoostedDeepTauID tagger is better that old BDT-based taggers with Run-3 data, however it is worse than for Run-2 data. It seems that degradation is similar as for the regular DeepTauID v2.5. Probably a dedicated training with Run3 data would be better, but it is not on the place so current one is the best available.